inc/functions/formatting.php

```diff
@@ -20,6 +20,35 @@ function rocket_clean_exclude_file( $file ) {
 	return wp_parse_url( $file, PHP_URL_PATH );
 }
 
+/**
+ * Clean Never Cache URL(s) bad wildcards
+ *
+ * @since 3.4.2
+ * @author Soponar Cristina
+ *
+ * @param  string       $path URL which needs to be cleaned
+ * @return bool\string  false if $path is empty or cleaned URL
+ */
+function rocket_clean_wildcards ( $path ) {
+	if ( ! $path ) {
+		return false;
+	}
+
+	$path_components = explode( '/', $path );
+    $arr             = array(
+            ".*"   => "(.*)" ,
+            "*"    => "(.*)" ,
```

Comment:
I suppose we could also add `'(*)' => '(.*)',`.
Also, phpcs won't be happy.

Reply:
Nice use of `strtr()` 👍

Reply:
Added also: `'(*)' => '(.*)' rule`

Applied to CSS/JS files exclusions, User Agents exclusion, Always Purged URLs, and CDN exclusions

Reply:
Last changes about styling:
- `$arr             = array(`: short array syntax please.
- `".*"   => "(.*)" ,`: no space character before the last `,` for the 4 lines.

Reply:
Coding style changes applied. Also I changed all array() into [] shorthand. It's confusing to see array()